ironic/drivers/modules/drac/raid.py

```diff
@@ -761,13 +910,27 @@ def delete_configuration(self, task):
         :raises: DracOperationError on an error from python-dracclient.
         """
         node = task.node
+        controllers = list()
+        drac_raid_controllers = list_raid_controllers(node)
+        for cntrl in drac_raid_controllers:
+            if is_raid_controller(node, cntrl.id):
```

Comment:
I think we have a bug in the existing code.  is_raid_controller will return False for BOSS cards, which is incorrect.  The end result is that the RAID1 on BOSS cards will not be deleted by this code.  Has the code below been tested on BOSS cards?

Reply:
No as of now we are testing only on Raid Controllers (H730 perc) which is present on 13G (R7)stamp

Reply:
This patch resolves the issue: https://review.opendev.org/#/c/653850/